apps/studio/src/app/page.tsx

```diff
@@ -1,10 +1,17 @@
 "use client";
 
+import { useRouter } from "next/navigation";
+import { useEffect } from "react";
 import { Toaster } from "react-hot-toast";
 import { Tooltip } from "react-tooltip";
 import { MainPage } from "~/pageComponents";
 
 export default function Page() {
+	const router = useRouter();
+	useEffect(() => {
+		router.push("https://codemod.com/studio");
```

Comment:
how about storing all urls as constants in single file to avoid strings?
i.e. in file `constants/urls.ts`

Reply:
sure, agree